exporter/exporter.go

```diff
@@ -85,6 +85,16 @@ func LoadMeasuresAndValueSets(measures []byte, valueSets []byte) {
 
 // GenerateCat1 generates a cat1 xml string for export
```

Comment:
It's pretty typical in programming to write a public function that is really a wrapper around a private function, I don't think it gets more explicit.